relation_engine_server/test/test_api_v1.py

```diff
@@ -96,20 +99,20 @@ def test_request(self, url=None, params=None, data=None, headers=None, method='g
 
     def test_root(self):
         """Test root path for api."""
-        resp = requests.get(URL + '/').json()
-        self.assertEqual(resp['arangodb_status'], 'connected_authorized')
-        self.assertTrue(resp['commit_hash'])
-        self.assertTrue(resp['repo_url'])
+        resp_json = requests.get(URL + '/').json()
```

Comment:
Consistent naming for responses -- if it is a response object, call it `resp`; if it is the deJSONified response body, call it `resp_json`.